src/main/java/com/github/manolo8/darkbot/utils/CaptchaAPI.java

```diff
@@ -0,0 +1,16 @@
+package com.github.manolo8.darkbot.utils;
+
+public interface CaptchaAPI {
+    String TWOCAPTCHA_PATH = "lib/captchasolver.jar";
+    CaptchaAPI INSTANCE = ReflectionUtils.createInstance("eu.darkbot.captcha.CaptchaSolver", TWOCAPTCHA_PATH);
+    static CaptchaAPI getInstance(){
+        return INSTANCE;
+    }
+
+    /**
+     * Pass a website page and it will parse the g-site-key and solve using external service.
+     * @param webpage a page that has been consumed into a string
+     * @return A solved g-captcha-response key from 2Captcha Solver Service
+     */
+    String solveCaptcha(String webpage);
```

Comment:
```suggestion
     * Create solving captcha parameters for the provided webpage html string.
     * @param the url this webpage was obtained from
     * @param webpage as an HTML string
     * @return form parameters to include as captcha response solution
     */
    Map<String, String> solveCaptcha(URL url, String webpage);
```
Passing in the URL allows you to send it over to the resolver service, which is needed, instead of hard-coding the url in the implementation

For the response map, this implementation can just be, a:
`Collections.singletonMap("g-captcha-response", response);`

However other types of captchas may use different responses (or more of them)